README.md

```diff
@@ -53,22 +64,43 @@ Then go to <http://localhost:4000/>.
 
 ## Run with docker
 
-Videoroom demo provides a `Dockerfile** that you can use to run videoroom application yourself without any additional setup and dependencies.
+Videoroom demo provides a `Dockerfile` that you can use to run videoroom application yourself without any additional setup and dependencies.
 
 ### To run:
 
-Default environmental variables are available in .env file, you can adjust it to your needs.
+**IMPORTANT** If you intend to use TLS remember that setting paths in `.env` file is not enough. Those paths will be used inside the docker container therefore besides setting env variables you will need to mount those paths to the docker container on your own.
+
+Default environmental variables are available in `.env` file. To run Membrane Videoroom, firstly you have to set up integrated TURNs environments there. Add these 2 lines to this file:
+```bash
+INTEGRATED_TURN_PORT_RANGE=50000-50050
+INTEGRATED_TURN_IP={IPv4 address of one of your network interfaces}
+```
+`INTEGRATED_TURN_PORT_RANGE` describes the range, where TURN servers will try to open ports. The bigger range is, the more users server will be able to handle. If you will use `--network=host` option when running `docker`, there is no need to set the value of this variable. Otherwise, remember to publish range of ports on `UDP` traffic. 
+`INTEGRATED_TURN_IP` is the IP address, on which TURN servers will listen. You can get it from the output of `ifconfig` command. To make the server available from your local network, you can set it to address like `192.168.*.*`.
 
-**IMPORTANT** If you intend to use TLS remember that setting paths in .env file is not enough. Those paths will be used inside docker container therefore besides setting env variables you will need to mount those paths to docker container on your own.
+To build docker image from source, run
+```bash
+$ docker build  -t membrane_videoroom .
+```
 
+To start a container with this image, run 
+```bash
+$ docker run --network=host --env-file .env membrane_videoroom
+```
+or
 ```bash
-docker run -p 4000:4000 --env-file .env membraneframework/demo_webrtc_videoroom_advanced:latest
+$ docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp --env-file .env membrane_videoroom
```

Comment:
Instead of suggesting editing the `.env` file here, you should rather add `-e` for it